src/components/Logo/Logo.story.jsx

```diff
@@ -0,0 +1,16 @@
+/* eslint-disable import/no-extraneous-dependencies */
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { select, text } from '@storybook/addon-knobs';
+import Logo from './Logo';
+
+const stories = storiesOf('Branding', module);
+
+stories.add('Logo', () => (
+  <Logo
+    title={text('Title', 'Data Tracker')}
+    kind={select('Kind', ['simple', 'full'], 'full')}
+    size={select('Size', ['small', 'medium', 'large'], 'medium')}
```

Comment:
would add the pixel size to these for documentation purposes like `small (20 x20px)`